index.html

```diff
@@ -204,6 +204,18 @@ <h3 class="sub1">Multi-Post Stories</h3>
          </div>
       </div>
    </section>
+   <section class="contact">
+      <h2 class="contact-head">CONTACT US</h2>
+      <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <div class="form">
+         <form method="post" action="https://formspree.io/f/mknyrjkd" class="form">
+            <input type="text" name="name" id="name" placeholder="Text Input" required maxlength="30" class="input">
+            <input type="email" name="email" required class="input" value="jeremy@gmail.com">
+            <textarea placeholder="Enter Text Here" maxlength="500" class="textarea"></textarea>

```

Comment:
- [ ] The form is not supposed to be submitted if one or more of the fields is empty. Right, you are missing the required attribute from the message field. Please add the required attribute in your message field.

Reply:
All good now 👍🏽